recipes/funcsigs/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set name="funcsigs" %}
+{% set version="1.0.2" %}
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+    - python-{{ name }} {{ version }}
+
+test:
+  imports:
+    - funcsigs
+
+about:
+  home: http://funcsigs.readthedocs.org
+  license: Apache 2.0
+  summary: Python function signatures from PEP362 for Python 2.6, 2.7 and 3.2+.
```

Comment:
Copied the old metadata. Though this is a bit annoying. Wonder if there is a way we don't even have to copy.